package/skylark_download_daemon/Config.in

```diff
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_SKYLARK_DOWNLOAD_DAEMON
+	bool "skylark_download_daemon"
+	select BR2_PACKAGE_LIBCURL
+	select BR2_PACKAGE_CZMQ
+	select BR2_PACKAGE_LIBSBP
+	select BR2_PACKAGE_LIBSBP_ZMQ
+  select BR2_PACKAGE_CA_CERTIFICATES
```

Comment:
TODO: Fix formatting. WTF.
TODO: Do all of these selections need to be here? Can just BR2_PACKAGE_LIBSKYLARK be included?